.ci/openshift-ci/run_smoke_test.sh

```diff
@@ -57,14 +57,19 @@ else
 fi
 
 info "Wait for the HTTP server to respond"
-rm -f hello_msg.txt
-waitForProcess 60 1 "curl '${host}:${port}${hello_file}' -s -o hello_msg.txt"
-
-grep "${hello_msg}" hello_msg.txt > /dev/null
-test_status=$?
-if [ $test_status -eq 0 ]; then
+if waitForProcess 60 1 "curl '${host}:${port}${hello_file}' 2>/dev/null | grep -q '$hello_msg'"; then
```

Comment:
The difference is described in commit message. Previously we waited for any successful curl reply which, quite often, is actually an error message from OCP saying the service is not yet ready (but still returns OK 200). While this version waits for the output to be the expected string so even successful curl reply that does not contains such message will be ignored and retried later.

Reply:
Ok, I missed the double-quotes around the `curl | grep` pipeline... 

An option to avoid confusion and make this easier to review would be to do something like :
```
check_command="curl '${host}:${port}${hello_file}' 2>/dev/null | grep -q '$hello_msg'"
if waitForProcess 60 1 "$check_command"; then
```

Reply:
Sure, I can change that in v2